src/client/src/ui/analytics/views/analyticsView.jsx

```diff
@@ -43,12 +44,24 @@ export default class AnalyticsView extends ViewBase {
     if (!model.isAnalyticsServiceConnected)
       return <span />;
 
+    let browser = new userAgentParser().getBrowser().name;
```

Comment:
Yes, i was going to do that today. now that we've got a bit more time to do it properly